docs/Development/ReleaseNotes.md

```diff
@@ -63,11 +63,11 @@ This is a major update from VIC 4. The VIC 5.0.0 release aims to have nearly ide
 
 10. Testing and Continuous Integration ([GH#190](https://github.com/UW-Hydro/VIC/pull/190))
 
-	A comprehensive testing platform has been implemented and is available for public use along with the VIC model. A small subset of the test platform is run on [Travis-CI](https://travis-ci.org/UW-Hydro/VIC), which facilitates for continuous integration of the VIC test platform. More information on the test platform is [here](http://vic.readthedocs.org/en/develop/Development/Testing/)
+	A comprehensive testing platform has been implemented and is available for public use along with the VIC model. A small subset of the test platform is run on [Travis-CI](https://travis-ci.org/UW-Hydro/VIC), which facilitates for continuous integration of the VIC test platform. More information on the test platform is [here](Testing.md)
```

Comment:
Just "facilitates continuous integration" not "facilitates for continuous integration"